src/test/java/de/tum/in/www1/artemis/exercise/ExerciseUtilService.java

```diff
@@ -94,13 +94,25 @@ public class ExerciseUtilService {
     @Autowired
     private ConversationUtilService conversationUtilService;
 
+    /**
+     * Sets the max points of an exercise to 100 and sets bonus points to 10. IncludedInOverallScore of the exercise is set to INCLUDED_COMPLETELY.
+     *
+     * @param exercise The exercise to be updated.
+     * @return The saved exercise.
+     */
     public Exercise addMaxScoreAndBonusPointsToExercise(Exercise exercise) {
         exercise.setIncludedInOverallScore(IncludedInOverallScore.INCLUDED_COMPLETELY);
         exercise.setMaxPoints(100.0);
         exercise.setBonusPoints(10.0);
         return exerciseRepo.save(exercise);
     }
 
+    /**
+     * Adds grading instructions from three different grading criteria to an exercise. One criterion is empty.
+     *
+     * @param exercise The exercise to which the grading instructions should be added.
+     * @return The list of grading criteria.
+     */
```

Comment:
Should we write here more about what kind of instructions are added? Like `default` or `test`?

Reply:
I'm not exactly sure what you mean ba `default` and `test` instructions. If you mean the variable names `testCriterion` and `testCriterion2`, I looked into it and saw that this is only based on the title of the corresponding criterion. I don't think we actually have different types of criteria and instructions. I now renamed the one JavaDoc that it does not mention `empty criterion`.